### ISSUE TITLE
reviewable.io is horrible experience 

### ISSUE THREAD

**Issue by dz0ny** [ubuntu/snappy-playpen#176]
- The > github sync creates too much clutter
- New push does not clear comments for that part of code
- You have 6(discussing,satisfied,blocking,unresolvable,solvable)!! options on how to respond to code comment, rly?
- Confusing UI, aka they use "reactive" interface but you have hidden publish somewhere on page :)
- As PR submitter you have to mark each file as viewed(http://i.imgur.com/qiIMeC0.png) in order to make comment thread "resolvable"
- Each button has 3 options (primary,help,butterflies something)
- Comments are gone after they have been resolved? I like to piggyback on closed PR to see what was discussed about that part of code at that time... and you cant see them after reviewer marked them as resolved

TLDR; Confusing

**Comment by dholbach** [comment]
```irc
<dz0ny> Is that just me https://github.com/ubuntu/snappy-playpen/issues/176
<dholbach> dz0ny, what if you open https://reviewable.io/reviews/ubuntu/snappy-playpen/166?
<dholbach> I agree that reviewable clutters up the normal pull request views
<dholbach> but I feel that it makes it much easier to keep track of what exactly still needs to be resolved in a list of PRs where some get stale and sit there and you never know which one you still need to look at
<dholbach> https://reviewable.io/reviews#- is a very clear overview for me
```

Can everybody else please comment on this as well, so we can do a check-up after a few weeks of use and see how we all feel?

**Comment by dplanella** [comment]
After having tried to use it, I must say I feel the same. For our purposes, I find doing reviews on the github interface much simpler. It might be just me, but I just don't quite grasp the Reviewable interface. Also, to my irritation, after having sent a pull request and having spent a few minutes writing the PR's description, reviewable overwrote my message (to add the Reviewable button snippet) while I was typing it, thus losing all that work.

I can be convinced that it's useful, though. It's just that for now, I find the regular PR web interface much easier to use.

**Comment by dholbach** [comment]
The reason @didrocks and I started looking for alternative ways to review PRs was that many of them got stale and were waiting for updates from the submitters. Thus, the list of PRs (https://github.com/ubuntu/snappy-playpen/pulls) grew and unless you took the time to go through them all (regularly), we couldn't make any progress.

I agree that using normal PRs on Github now with Reviewable sending funky messages (that only make sense to Reviewable itself) is weird and almost non-sensical.

After doing a few reviews on Reviewable though, I must admit that I'm very happy. It supports a number of things:

 - mark a file as reviewed and be able to review just the stuff which changed since last time in one glance, without clicking a lot
 - indicate if a change you comment on is a nice-to-have or a blocker
 - formalise reviews where you have many reviewers
 - have an overview over which reviews are still outstanding, how many hunks were updated, etc

All in all, I'm very happy with the workflow, but if this is too cumbersome to use, we can look for alternatives. I seem to remember there were other services which could be hooked up with GitHub too.

**Comment by didrocks** [comment]
Yeah, I can totally see why people would find it confusing (and it is at first time). I really like the features that Daniel mentionned. Especially after some changes on a PR (with additional commits) are done. You are not reviewing again what didn't change and what you already acked.

As we have a lot of reviews with force and back, this is the number 1 feature to me. If something more simple for people exists, I'm opened to this as well.

**Comment by tsimonq2** [comment]
I agree with @dz0ny. I **strongly** dislike the comments it spams. Yes, the interface might be nice for the reviewers, but for me as a contributor, it is **very** annoying.

It also doesn't update the diffs, which @dz0ny mentioned, and that is annoying. Why not use the GitHub inline comments?

In my honest opinion, we need a different solution.

**Comment by popey** [comment]
Signed in on my phone on holiday to express how painful I find the whole reviewable process. Echoing what others have said. It has a maddeningly busy ux and Sammy comment behaviour that actively discourages me from getting involved.

**Comment by popey** [comment]
*spammy

**Comment by didrocks** [comment]
So, we did disable reviewable and moved to codereviewhub.
We still have (as reviewable):
- it tracks comment on old diff
- you can +1/-1 comments

The advantages are:
- it's entirely in the github PR view (no more separate website)
- inline comments create tasks automatically that you can approve later on (once fix)
- it's using the old diff feature from github

From some tests on other projects, it seems to be what we really want. So, I disabled reviewable and enable the new tool (note that the old PR still have the reviewable links and so, a list of tasks in codereviewhub).

New PR should now be way cleaner and understandable to everyone :)
I'm thus closing this issue.
